packages/zoe/src/types.js

```diff
@@ -0,0 +1,218 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="ses"/>
+
+/**
+ * @typedef {any} TODO Needs to be typed
+ * @typedef {string} Keyword
+ * @typedef {{}} InstallationHandle
+ * @typedef {Object<string,Issuer>} IssuerKeywordRecord
+ * @typedef {Object<string,Brand>} BrandKeywordRecord
+ * @typedef {Object<string,Payment>} PaymentKeywordRecord
+ * @typedef {Object<string,Promise<Payment>>} PaymentPKeywordRecord
+ * @typedef {Object} SourceBundle
+ * @property {string} source
+ * @property {string} sourceMap
+ * @property {string} moduleFormat
+ */
+
+/**
+ * @typedef {Object} ZoeService
+ * Zoe provides a framework for deploying and working with smart contracts. It
+ * is accessed as a long-lived and well-trusted service that enforces offer
+ * safety for the contracts that use it. Zoe has a single `inviteIssuer` for
+ * the entirety of its lifetime. By having a reference to Zoe, a user can get
+ * the `inviteIssuer` and thus validate any `invite` they receive from someone
+ * else.
+ *
+ * Zoe has two different facets: the public Zoe service and the contract facet
+ * (ZCF). Each contract instance has a copy of ZCF within its vat. The contract
+ * and ZCF never have direct access to the users' payments or the Zoe purses.
+ * The contract can only do a few things through ZCF. It can propose a
+ * reallocation of amount or complete an offer. It can also speak directly to Zoe
+ * outside of its vat, and create a new offer for record-keeping and other
+ * purposes.
+ *
+ * @property {() => Issuer} getInviteIssuer
+ * Zoe has a single `inviteIssuer` for the entirety of its lifetime.
+ * By having a reference to Zoe, a user can get the `inviteIssuer`
+ * and thus validate any `invite` they receive from someone else. The
+ * mint associated with the inviteIssuer creates the ERTP payments
+ * that represent the right to interact with a smart contract in
+ * particular ways.
+ *
+ * @property {(bundle: SourceBundle, moduleFormat?: string) => InstallationHandle} install
```

Comment:
```suggestion
 * @property {(bundle: SourceBundle) => InstallationHandle} install
```
install takes a single argument which is a bundle